hack/netpol/install.yml

```diff
@@ -4,15 +4,15 @@
 apiVersion: batch/v1
 kind: Job
 metadata:
+  namespace: kube-system
```

Comment:
namespace already present on L9

Reply:
auto merge issue, thanks for catching this